apps/api/src/app/organization/organization.controller.ts

```diff
@@ -154,4 +157,16 @@ export class OrganizationController {
       })
     );
   }
+
+  @Put('/')
```

Comment:
As we are just modifying part of the resource `organization`, the HTTP method needs to be `PATCH`.
I would also recommend to rename the method name to `renameOrganization` to not ocuppy the name for a future organization update functionality that updates the whole resource. Same for the DAL repository method `updateOrganization`.

Reply:
Personally, I don't really subscribe to the fact that we are having different endpoints to update part of organization resource. I feel it should be a single endpoint with the possibility of updating part of the resource or the whole resource, that is why I am using the `name` column in this case. I can update the method to `PATCH`., but I think we should still keep the name as is. what do you think  @p-fernandez ?

Reply:
You are right now we don't have those endpoints, but in the future it is an option that I would like to have it open. 👍🏻 

Regarding the naming, when I read the method's name I would expect inside implements the functionality to update any modifiable property of the organization resource: name, logo, branding, partnerConfigurations (https://github.com/novuhq/novu/blob/4ae361d5676f007ece261698fe3447fccb9d9acf/libs/dal/src/repositories/organization/organization.schema.ts#L6)
As that is not what's happening giving a more accurate name of what the method is doing helps to any developer or user that goes through the code.

Reply:
Correct.. but the command shows it. The payload has only a `name` property, so one can still know that only `name` is being updated.

We can update the name if and only if we just want to use this method to update only name and not touch it in the future, i.e, updating other part of the organization resource in the future.